routers/users.py

```
@@ -0,0 +1,189 @@
+
+        # [중요] 윈도우 역슬래시(\) 문제를 방지하기 위해 URL 형식(/)으로 통일
+        image_url = f"/static/uploads/{safe_filename}"
+
+    # 사용자 객체 생성 (비밀번호 해싱 필수)
+    new_user = {
+        "id": new_id,
+        "email": email,
+        "password": get_password_hash(password),
+        "nickname": nickname,
+        "profileImage": image_url,
+        "createdAt": datetime.now().isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)